src/index.html

```diff
@@ -3,75 +3,443 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Product</title>
-
-    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
     <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-      crossorigin="use-credentials"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     >
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
-
+    <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
-        <img src="./images/logo.png" alt="Moyo logo">
+      <a href="#home" class="header__logo">
+        <img
```

Comment:
no need to set them into separate strings if it'is not more than 80 sym or parametrs less than 3. chech it all the way